src/shared/store/actions/queryResource.ts

```diff
@@ -94,88 +81,88 @@ const queryResourcesFailedAction: ActionCreator<FailedQueryAction> = (
   type: QueryResourcesActionTypes.FAILED,
 });
 
-export const queryResources: ActionCreator<ThunkAction> = (
-  id: string,
-  org: Organization,
-  project: Project,
-  paginationSettings: PaginationSettings,
-  query?: List['query']
-) => {
-  return async (
-    dispatch: Dispatch<any>,
-    getState,
-    { nexusLegacy }
-  ): Promise<
-    | FilterFetchFulfilledAction<
-        QueryResourcesActionTypes.FULFILLED,
-        QueryResourcesFulfilledPayload
-      >
-    | FilterFetchFailedAction<QueryResourcesActionTypes.FAILED>
-  > => {
-    const Project = nexusLegacy.Project;
-    const filterKey = makeOrgProjectFilterKey(org, project);
-    const listState = (getState() as RootState).lists;
-    const targetWorkspace = listState && listState[filterKey];
-    const filterIndex = (targetWorkspace || []).findIndex(
-      (list: List) => list.id === id
-    );
-    const list: List | undefined =
-      (!!targetWorkspace && filterIndex >= 0 && targetWorkspace[filterIndex]) ||
-      undefined;
-    try {
-      if (!list) {
-        throw new Error(
-          `no list found with id ${id} inside project ${project.label}`
-        );
-      }
-      dispatch(queryResourcesFetchAction(filterIndex, filterKey));
-      const formattedQuery = makeESQuery(query);
-      const realProject = await Project.get(org.label, project.label);
-      const defaultElasticSearchView: ElasticSearchView = await realProject.getElasticSearchView();
-      const resources: PaginatedList<
-        Resource
-      > = await defaultElasticSearchView.query(
-        formattedQuery,
-        paginationSettings
-      );
-      const aggregationQuery = {
-        ...formattedQuery,
-        track_total_hits: true,
-        aggs: {
-          schemas: {
-            terms: {
-              size: 999,
-              field: '_constrainedBy',
-            },
-          },
-          types: {
-            terms: {
-              size: 999,
-              field: '@type',
-            },
-          },
-        },
-      };
-      const aggregationResponse: ElasticSearchViewAggregationResponse = await defaultElasticSearchView.aggregation(
-        aggregationQuery
-      );
-      const schemas = aggregationResponse.aggregations.schemas.buckets.map(
-        ({ doc_count, key }) => ({ key, count: doc_count })
-      );
-      const types = aggregationResponse.aggregations.types.buckets.map(
-        ({ doc_count, key }) => ({ key, count: doc_count })
-      );
-      return dispatch(
-        queryResourcesFulfilledAction(
-          filterIndex,
-          filterKey,
-          resources,
-          schemas,
-          types
-        )
-      );
-    } catch (e) {
-      return dispatch(queryResourcesFailedAction(filterIndex, filterKey, e));
-    }
-  };
-};
+// export const queryResources: ActionCreator<ThunkAction> = (
```

Comment:
We can assume that this can go then? :laughing:

Reply:
yes!